pkg/cmd/roachtest/tests/mixed_version_c2c.go

```diff
@@ -142,7 +148,6 @@ type c2cMixed struct {
 }
 
 func (cm *c2cMixed) SetupHook(ctx context.Context) {
-	sourceChan := make(chan sourceTenantInfo, 1)
```

Comment:
then, in node 5's logs on dest:
```
* ERROR: ERROR: cannot load certificates.
* Check your certificate settings, set --certs-dir, or use --insecure for insecure clusters.
*
* server startup failed: failed to start server: problem using security settings: no certificates found; does certs dir exist?
```

Reply:
Hey, I should have added a comment about that. The reason the source needs to start first is because roachprod's support for multiple CRDB clusters in a single roachprod cluster is very finicky. Due to the way certificate management works, it's important for the CRDB cluster that n1 is a part of to be started first, otherwise you get that error.

It's really not ideal and we should properly support multiple CRDB clusters in roachprod (roachprod was designed with the assumption that `roachprod cluster == crdb cluster`), but that's a tricky task and I didn't want to block this work on that. So I think we need to keep that logic for now.

Reply:
Created #129318.

Reply:
ah, no worries! I'm actually not sure how urgent https://github.com/cockroachdb/cockroach/issues/129318 is, since the workarounds for PCR (and LDR in the future) are not that painful.